src/Uintra20/Content/uintra20/src/app/ui/pages/uintra-groups/create/uintra-groups-create-page.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { finalize } from 'rxjs/operators';
+
+@Component({
+  selector: 'uintra-groups-create-page',
+  templateUrl: './uintra-groups-create-page.html',
+  styleUrls: ['./uintra-groups-create-page.less'],
+  encapsulation: ViewEncapsulation.None
+})
+export class UintraGroupsCreatePage {
+  data: any;
+  title: string = "";
+  description: string = "";
+  files: any[] = [];
+  isShowValidation: boolean = false;
+  inProgress: boolean = false;
+
+  constructor(
+    private route: ActivatedRoute,
+    private http: HttpClient
+  ) {
+    this.route.data.subscribe(data => this.data = data);
+  }
+
+  get isSubmitDisabled() {
+    return this.inProgress;
+  }
+
+  ngOnInit() {
+
+  }
+
+  onUploadSuccess(fileArray: Array<any> = []): void {
+    this.files.push(fileArray);
+  }
+
+  onFileRemoved(removedFile: object) {
+    this.files = this.files.filter(file => file[0] !== removedFile);
+  }
+
+  onSubmit() {
+    if (this.validate()) {
+      this.inProgress = true;
+
+      const groupCreateModel = {
+        title: this.title,
+        description: this.description,
+        newMedia: this.getMediaIdsForResponse(),
+        media: null
+      }
+
+      this.http.post('/ubaseline/api/Group/Create', groupCreateModel).pipe(
+        finalize(() => this.inProgress = false)
+      ).subscribe(res => {});
```

Comment:
remove unused code

Reply:
it will be used once we have details page to redirect to